src/app/dashboard/Missions/Missions.jsx

```diff
@@ -52,36 +53,36 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const ViewButtons = ({ classes, missionsView }) => {
+const ViewButtons = ({ classes, inDone, inPlanning, inProgress, inProposed, missionsView }) => {
   const history = useHistory();
   const ordered = [
     {
       view: "inProposed",
-      text: "Proposed",
+      text: "Proposed (" + inProposed.length + ")",
       onClick: () =>
         history.replace({
           search: _.setQueryParam("view", "inProposed"),
         }),
     },
     {
       view: "inPlanning",
-      text: "Planning",
+      text: "Planning (" + inPlanning.length + ")",
       onClick: () =>
         history.replace({
           search: _.setQueryParam("view", "inPlanning"),
         }),
     },
     {
       view: "inProgress",
-      text: "In Progress",
+      text: "In Progress (" + inProgress.length + ")",
       onClick: () =>
         history.replace({
           search: _.setQueryParam("view", "inProgress"),
         }),
     },
     {
       view: "inDone",
-      text: "Done",
+      text: "Done (" + inDone.length + ")",
```

Comment:
while you have the file back open would be good to see these as template literals instead

```js
text: `Done (${inDone.length})`
```